nexa/gguf/llama/llava_cpp.py

```diff
@@ -33,7 +33,7 @@
 from nexa.gguf.lib_utils import load_library
 
 # Specify the base name of the shared library to load
-_libllava_base_name = "llava"
```

Comment:
Did we check that other OS all have _shared suffix? I though the problem was only on windows?

Reply:
I have checked that all os have _shared suffix